extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -18,17 +18,19 @@
  * @fileoverview Helper for amp-story rendering of page-attachment UI.
  */
 import {LocalizedStringId} from '../../../src/localized-strings';
+import {computedStyle, setImportantStyles} from '../../../src/style';
 import {getLocalizationService} from './amp-story-localization-service';
+import {getRGBFromCssColorValue, getTextColorForRGB} from './utils';
 import {htmlFor, htmlRefs} from '../../../src/static-template';
 import {isExperimentOn} from '../../../src/experiments';
-import {setImportantStyles} from '../../../src/style';
+import {AttachmentTheme} from './amp-story-page-attachment';
 
 /**
  * @enum {string}
  */
-const AttachmentTheme = {
-  LIGHT: 'light', // default
-  DARK: 'dark',
+ const ctaAccentElement = {
```

Comment:
```suggestion
const CtaAccentElement = {
```
And plz replace (first letter uppercase) below :))